external-crates/move/crates/move-analyzer/src/symbols.rs

```diff
@@ -2729,7 +2730,17 @@ impl<'a> ParsingSymbolicator<'a> {
                     }
                 })
             }
-            MP::Name(_, chain) => self.chain_symbols(chain),
+            MP::Name(_, chain) => {
+                self.chain_symbols(chain);
+                assert!(self.current_mod_ident_str.is_some());
+                let Some(mod_defs) = self
+                    .mod_outer_defs
+                    .get_mut(&self.current_mod_ident_str.clone().unwrap())
+                else {
+                    return;
+                };
+                mod_defs.untyped_defs.insert(chain.loc);
```

Comment:
untested but I think you can `if let` this for simplicity, something like:

```rust
        if let Some(mod_defs) = self.mod_outer_defs.get_mut(current_mod) {
            mod_defs.untyped_defs.insert(chain.loc);
        }
```

Reply:
Done!